AnyKeyDetector.cs

```diff
@@ -0,0 +1,15 @@
+﻿using UnityEngine;
+
+public class AnyKeyDetector : MonoBehaviour
+{
+    public LevelLoader ll;
```

Comment:
Name variables with meaningful names.
It is used to write it after the type, but if the type is very long it is desirable to use an abbreviated name.

```suggestion
    public LevelLoader levelLoader;
```